packages/view/src/boot/hyemimi/BarChart.tsx

```diff
@@ -0,0 +1,117 @@
+import React, { useEffect, useRef, useState } from "react";
+import * as d3 from "d3";
+
+interface DataItem {
+  source: string;
+  target: string;
+  value: number;
+}
+
+const csvPath = "/boot/raw.csv";
+
+const BarChart: React.FC = () => {
```

Comment:
React.FC는 `children이 암시적으로 포함`되어있을 수 있어서 최대한 사용을 지양하자고 알고 있는데요! 
children을 명백히 받는 경우가 아니라면
```suggestion
const BarChart = () => {
```
위와 같이 일반적으로 가도 될 것 같습니다.

저는 보통 children이 필요할 때 타입을 따로 설정해두고 잘 안쓰는 편이라ㅎㅎ 스타일 차이일 수도 있으니 일단 의견 달아봅니다:)

Reply:
의견 감사합니다 ㅎㅎ
React.FC가 children을 props 타입에 암시적으로 포함시킨다는 건 이번에 처음 알게 되었습니다 ..!
컨벤션 상으로도 적절하지 않은 것 같아 해당 부분은 수정하여 반영하겠습니당